src/box/lua/watcher.c

```diff
@@ -219,6 +209,36 @@ lbox_broadcast(struct lua_State *L)
 	return rc == 0 ? 0 : luaT_error(L);
 }
 
+/**
+ * Allows to broadcast events, the key of which starts with box.*.
+ */
+static int
+lbox_broadcast_internal(struct lua_State *L)
+{
+	int top = lua_gettop(L);
+	if (top != 1 && top != 2)
+		return luaL_error(L, "Usage: box.broadcast(key[, value])");
```

Comment:
Nit: I guess it should be `box.internal.broadcast` instead of `box.broadcast`.

Reply:
Fixed